echo-web/config/echo.yml

```diff
@@ -32,8 +32,6 @@ webhooks:
 
 scheduler:
   enabled: ${services.echo.cron.enabled:true}
-  cron:
-    timezone: ${global.spinnaker.timezone:America/Los_Angeles}
```

Comment:
Ooh no I was thinking about this wrong, you're 100% right. ⌛😱 We can still take it out of the `/halconfig/echo-scheduler` since that is appended to the base web config, right?

Reply:
Yup, exactly, this looks good!